### DISCUSSION TITLE
How to set a scale factor in terms of distribution in population simulations?

### DISCUSSION THREAD

**Discussion by xiangetang** [Open-Systems-Pharmacology/Forum#1750] [Questions & Problems]
In the individual simualtion, we can set a scale factor value to adjust the shape of distribution of concentration-time curve. I have established my individual PBPK model (Adult), but when I use it to simulate in population (Children), I found that I cann't use the scale fator in the population simulation?
Another question:  can the scale factor be optimized using "Parameter identification" tool?
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/98267394/a4ed1754-6e48-43ae-85ed-3afe5dc2c0aa)

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/98267394/0aa36be0-ec9b-4a42-ae86-8353255cf354)

Any suggestion would be appreciated! thanks in advance!

**Comment by Yuri05** [comment]
The _Scale_ option here is not a model parameter - it just multiplies all shown parameters by the given value. 
If you want to change the value of Fraction unbound in plasma - you should set the fu reference value (which also can be optimized in a parameter identification in individual simulations)

![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/a0992d39-6b43-4d5d-8a66-f6a08c958e64)

**Reply by xiangetang** [reply]
Thanks very much, Yuri05!

The fu can usually be tested accurately in-vitro,via such as plasma-binding experiments，so fu tend to be input as it was. If there are some systematic-deviations of PK-sim in predicting  distrubition using phsico-chemistry, scale factor can be used to correct this systematic-deviation. However, scale factor cann't be found in population simulation.

What do you have any good suggestions for poor fit of distribution? thanks!

**Reply by Yuri05** [reply]
You can find/modify the plasma protein scale factor in a population:
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/7995678f-5da8-485a-8533-58108091d522)

> do you have any good suggestions for poor fit of distribution?

A poor fit of the distribution phase can occur for various reasons. Here are some suggestions to improve the fit of your distribution phase in PK-Sim:

1. **Review Input Parameters**: Ensure that all input parameters are accurate and reflect the drug's characteristics and the population being studied. Parameters such as lipophilicity (LogP), pKa, plasma protein binding, and molecular weight can significantly affect distribution.

2. **Adjust Partition Coefficients**: The partition coefficients determine the distribution of the drug into different tissues. You may need to adjust these coefficients based on observed data or literature values for similar compounds.

3. **Refine Tissue Composition Parameters**: Adjust the composition of tissues in terms of water, lipid, and protein content, as this can affect the volume of distribution and the drug's tissue affinity.

4. **Use Advanced Distribution Models**: If the standard distribution models do not fit well, consider using more advanced models that can account for specific distribution phenomena (e.g., permeability-limited distribution or active transport mechanisms).

5. **Incorporate Enterohepatic Circulation**: If the drug undergoes enterohepatic circulation, this process should be included in the model as it can affect both the distribution and elimination phases.

6. **Consider Disease States or Special Populations**: If modeling a special population or disease state, make sure to adjust the physiological parameters accordingly, as these can have a significant impact on drug distribution.

7. **Account for Metabolite Data**: If metabolites are significant and have different distribution properties than the parent compound, include them in the model.

8. **Model Optimization and Sensitivity Analysis**: Use the optimization and sensitivity analysis tools within PK-Sim to identify which parameters have the most significant impact on the distribution phase and adjust them within physiologically plausible ranges.

9. **Literature Comparison**: Compare your model's predictions with published data for the same or similar drugs. Discrepancies might provide insights into which aspects of the model need refinement.

10. **Expert Consultation**: If the distribution is still not fitting well after these adjustments, it may be beneficial to consult with a pharmacokinetic expert who can provide insights based on experience with similar compounds or models.

11. **Model Validation**: Validate your model with external data sets, if available, to ensure that the model can accurately predict the distribution in different scenarios or populations.

**Reply by xiangetang** [reply]
Yuri, thanks very much for your detailed reply. 
Another two related questions:

1. I did found e.g logP impact distribution significantly, while logP was determined by pharmaceutical experiment accurately, optimize logP seems not very reasonable?
2. If I develop my model in adult "individual", but I simulate senarios in children "population". Can the plasma scale factor be changed in children population?  Partition coeffiecient seems can only be changed by changing different calculation methods, or changing the value manually of a certain organ.

Any reply would be appreciated!

**Reply by Yuri05** [reply]
1. Still might make sense
2. yes.
